go.mod

```diff
@@ -14,14 +14,15 @@ require (
 	github.com/gorilla/websocket v1.5.0
 	github.com/grafana/xk6-redis v0.1.1
 	github.com/grafana/xk6-timers v0.1.2
-	github.com/grafana/xk6-websockets v0.1.3
+	github.com/grafana/xk6-websockets v0.1.4-0.20220909124839-39d27946bc0c
```

Comment:
Is this how you are going to leave it ? And then update the extenions and then k6 again?
Or shall we remove it(the extension) for this PR and then update it and readd.

Again in the first case as we will squash this PR the commit that the extension uses will be removed after merge.

Reply:
If we *remove* it from the source as well it will build